WcaOnRails/app/helpers/notifications_helper.rb

```diff
@@ -67,6 +67,16 @@ def notifications_for_user(user)
       }
     end
 
+    user.delegated_competitions.visible.over
+      .where("start_date >= ?", DelegateReport::REPORTS_ENABLED_DATE) # Don't notify about very old competitions without reports.
+      .includes(:delegate_report).where(delegate_reports: { posted_at: nil })
+      .each do |competition|
+        notifications << {
+          text: "#{competition.name} you delegated is pending a delegate report, please fill and post it.",
```

Comment:
The text here could probably be improved... maybe simply "The delegate report for #{competition.name} has not been submitted".